hybrasyl/Objects/User.cs

```diff
@@ -2370,6 +2573,11 @@ public void AssailAttack(Direction direction, Creature target = null)
                         {
                             target = user;
                         }
+                        var npc = obj as Merchant;
+                        if (npc != null)
+                        {
+                            target = npc;
+                        }

```

Comment:
Scratch that, this check cannot happen later on, as obj is held within the case statement itself. No change will occur here.